doc/release_notes.rst

```diff
@@ -41,6 +41,10 @@ Upcoming Release
 
 * Enable parallelism in :mod:`determine_availability_matrix_MD_UA.py` and remove plots. This requires the use of temporary files.
 
+* Added new feature that to base the electricity network on OpenStreetMap (OSM data) (PR https://github.com/PyPSA/pypsa-eur/pull/1079). Note that a heuristics based cleaning process is used for lines and links where electrical parameters are incomplete, missing, or ambiguous. Through ``electricity_network["base_network"]``, the base network can be set to "gridkit" (original default setting), "osm-prebuilt" (which downloads the latest prebuilt snapshot based on OSM data from Zenodo), or "osm-raw" which retrieves (once) and cleans the raw OSM data and subsequently builds the network. Note that this process may take a few minutes.
+
+* Voltage settings have been aggregated and are now directly read from the line type dictionary. Instead of ``electricity["voltages"]``, scripts have been updated to refer to ``lines["types"].keys()``.
```

Comment:
Due to the way the scenario management is set up, I would revert this.

Reply:
Done and updated release_notes.